src/components/application_manager/CMakeLists.txt

```diff
@@ -441,31 +167,25 @@ target_link_libraries("AMEventEngine" ${LIBRARIES})
 add_library("AMPolicyLibrary" ${POLICIES_MANAGER_SOURCES} )
 target_link_libraries("AMPolicyLibrary" ${LIBRARIES} AMEventEngine)
 
-add_library("AMHMICommandsLibrary" ${HMI_COMMANDS_SOURCES})
```

Comment:
is  there any new library replacing the AMHMICommandsLibrary? Or it is gone.

Reply:
@yang1070 now we have two libraries: sdl_rpc_plugin and rc_rpc_plugin 
https://github.com/smartdevicelink/sdl_core/pull/2196/commits/161858ef664190bf3165415256d35383c742a6e8#diff-4c2920e909ba977c71606bb3eb394317R61

https://github.com/smartdevicelink/sdl_core/pull/2196/commits/369c4959f84eec6abed8e9903d41195b0ee49191#diff-260fdedcfda831b8b98a5e3b6f69a097R61